og.module

```diff
@@ -112,6 +112,26 @@ function og_entity_access(EntityInterface $entity, $operation, AccountInterface
   if ($entity_type_id == 'node') {
     $node_access_strict = \Drupal::config('og.settings')->get('node_access_strict');
 
+    // Get the user's group membership.
+    $og_audience_field = $entity->get(\Drupal\og\OgGroupAudienceHelper::DEFAULT_FIELD);
```

Comment:
we already have an api function to get all the groups audience fields of a bundle. Let's use it here :)